frontend/ChapterList.vue

```diff
@@ -415,19 +420,51 @@
                   icon="phone"
                 ></b-input>
               </b-table-column>
-              <b-table-column field="Facebook" label="Facebook" v-slot="props">
-                <b-input
-                  type="text"
-                  v-model="props.row.Facebook"
-                  placeholder="Facebook"
-                  icon="facebook"
-                ></b-input>
-              </b-table-column>
               <b-table-column v-slot="props">
                 <b-button @click="deleteOrganizer(props.row)">
                   <b-icon icon="delete" type="is-danger"></b-icon>
                 </b-button>
               </b-table-column>
+              <template #detail="props">
+                <div style="display: flex; flex-wrap: wrap; gap: 10px">
```

Comment:
nit: i feel like there's a lot of space here. maybe gap-x 10px and gap-y 5px would look a little nicer

Reply:
just feels weird to me that there is more vertical space b/w these than there is b/w the "name email phone" fields and the top of this section.